frontend/src/components/RecentActivities/index.tsx

```diff
@@ -92,7 +102,19 @@ const RecentActivities = () => {
               </div>
               )}
             </TabPanel>
-            <TabPanel>Placeholder</TabPanel>
+            <TabPanel>
+              <FilesDocsTable
+              elements={filesDocsItems}
+              headers={fileDocstableHeaders}
+              />
+              <div className="empty-recent-activity-files-docs">
```

Comment:
Isn't necessary a check here to decide if this should be shown or not? (Just like line 95 of this file)

Reply:
fixed